### PR TITLE
Revive this mb

### PR DESCRIPTION
**Pull request by rylandely**

### CODE REVIEW

**Comment by robertzk** [comment]
I can't review this now, I will look at it shortly. In the meantime, can you fix CI?

**Comment by robertzk** [comment]
@rylandely lets look at this together

File: R/sure_independence_screen.r [hunk 1]
```diff
@@ -43,7 +43,7 @@ sure_independence_screen <- function(dataframe, ..., exclude = character(0),
       lower_count_bound = discretizer_params$lower_count_bound)
 
     # Raise an exception if the number of levels exceeds 100
-    if (any(violations <- vapply(dataframe, function(x) is.factor(x) &&
+    if (any(violations <- vapply(dataframe[,!names(dataframe) %in% exclude], function(x) is.factor(x) &&
```
Comment (robertzk) [inline R/sure_independence_screen.r]: Sneaky.

Comment (robertzk) [inline R/sure_independence_screen.r]: Use `drop = FALSE` or unary `[]`.

**Comment by rylandely** [comment]
@robertzk Still failing.

**Pull request closed without merging**